### ISSUE TITLE
Add Saraiki as Interface language

### ISSUE THREAD

**Issue by mpQadir** [Tatoeba/tatoeba2#3128]
Add Saraiki as interface language in Tatoeba Wiki, Tatoeba Term of Use and Tatoeba Website. Most work has been done on https://app.transifex.com/

**Comment by jiru** [comment]
Hello, welcome here and thank you for contributing to the translation of Tatoeba in Saraiki! :slightly_smiling_face: 

I can see that many strings have been translated in Saraiki on Transifex. But the default.pot part of "Tatoeba Website" is only 50% done, so I don’t think we can activate Saraiki on tatoeba.org already. I activated Saraiki on the development server https://dev.tatoeba.org/. You can use this special website to see how your translations look. The Saraiki language should appear under the name ﺱﺭﺎﺌﯿﮑﯾ (I copied this name from Wikipedia, please let me know if it should be changed). The development server is updated with new translations from Transifex every 10 minutes.

The next step is to continue translating the UI in Saraiki until default.pot is mostly done (ideally 100% but 80 or 90% is also acceptable). Refer to the [UI translation guide](https://en.wiki.tatoeba.org/articles/show/interface-translation) if you haven’t. Once this is done, let us know and we will activate Saraiki on https://tatoeba.org/.

The "Tatoeba Terms of Use" part is low priority because, well, people rarely read terms of use, but you can still translated it if you want.

As for the "Tatoeba Wiki" part, there is no need to translate it until a wiki is set up for Saraiki.

**Comment by mpQadir** [comment]
Thanks. Name ok

**Comment by mpQadir** [comment]
@jiru Dear 80% of default.pot has been translated into Saraiki.  Please activate Saraiki on https://tatoeba.org/.

**Comment by jiru** [comment]
@mpQadir Thanks for your hard work! :muscle: Saraiki has been activated on tatoeba.org. If you can, feel free to translate 100% of default.pot.
